src/components/CustomClient.js

```diff
@@ -3,18 +3,20 @@ import { Form, Message } from 'semantic-ui-react';
 
 
 /**
- * Propagate data from JSON object.
- * @callback load
- * @param {object} toLoad - Parsed JSON object to use.
- * @todo Change to just 'load'? Change this component
- *     to a more generic `<Load>`.
+ * @typedef {function} load Propagate data from JSON object.
+ * @callback
+ * @param {object} toLoad Parsed JSON object.
```

Comment:
The way it was before (without the `@typedef`) was ok! Unless you're trying to make it more clear that it defines a type?

And also, we can change the name here and in the type for `props.load` below to `CustomClient~load` so that the documentation gets grouped with the `CustomClient` class in the output. And/or we could group them in a module by putting `@module` at the top. Getting things out of the global namespace should be a goal. Maybe not necessary for this PR though.

Reply:
@ethanbb : Does that cover what's going on here? Does this PR feel ready to merge as it is, or are there more issues to iron out?

Reply:
Sorry to belabor this but I'm pretty sure the old syntax produces the exact same output, including the param type and description. Were you seeing something different?

I'll approve it though since this is not such a big deal!

Reply:
It's a good question, I think. I may be misunderstanding it. I'll try pinging you real time to clear this up.